src/storage/context/StorageExpressionContext.h

```diff
@@ -15,19 +15,18 @@
 namespace nebula {
 namespace storage {
 
-/*
-StorageExpressionContext supports both read value from a RowReader, or user set
-value by `setTagProp` and `setEdgeProp`.
-
-If we need to read value from the RowReader, be sure to set the reader by
-calling `reset`. For now, it only supports read from **one row**. So the reader
-is either a row of vertex or a row of edge. This mode is used in GetNeighbors at
-present.
-
-If we need to read value from a user defined place, just set the related value
-by `setTagProp` and `setEdgeProp`. Be sure about not pass the RowReader by
-`reset`.
-*/
+/**
+ * @brief StorageExpressionContext supports both read value from a RowReader, or user set
+ * value by `setTagProp` and `setEdgeProp`.
+ * If we need to read value from the RowReader, be sure to set the reader by
+ * calling `reset`. For now, it only supports read from **one row**. So the reader
+ * is either a row of vertex or a row of edge. This mode is used in GetNeighbors at
+ * present.
+ * If we need to read value from a user defined place, just set the related value
+ * by `setTagProp` and `setEdgeProp`. Be sure about not pass the RowReader by
+ * `reset`.
+ *
+ */
 class StorageExpressionContext final : public ExpressionContext {
```

Comment:
You could check whether the method is used in this class. IIRC, some methods related with index is outdated now, you could just delete them

Reply:
`indexCols()` removed after check.